Source/hydro/Castro_ctu.cpp

```diff
@@ -46,7 +46,7 @@ Castro::consup_hydro(const Box& bx,
 
     Real volinv = 1.0 / vol(i,j,k);
 
-    update(i,j,k,n) = update(i,j,k,n) +
+    update(i,j,k,n) = update(i,j,k,n) + dt *
```

Comment:
Can you change the name of this variable now to e.g. `U_new`

Reply:
updated